content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -2,135 +2,131 @@
+:guilabel:`Add a line`, which opens a new pricelist rules form.
 
-However, the :guilabel:`Time-based rules` tab and :guilabel:`Configuration` tab are always the same,
-regardless of the chosen :guilabel:`Pricelists` setting.
+Then, select whether to apply this set of rules to a :guilabel:`Product` or
+:guilabel:`Category`.
 
-Multiple prices per product
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+From here, there are several configuration options:
 
-With the :guilabel:`Multiple prices per product` setting activated, the :guilabel:`Price Rules` tab
-on pricelist forms provides the option to add specific products, with a specific price, to a
-pricelist.
+- :guilabel:`Product`: If selected in the :guilabel:`Apply To` field, use this field to choose one
+   or more products to which this pricelist will apply.
+- :guilabel:`Category`: Select one or more product categories to which this pricelist will apply.
+- :guilabel:`Price Type`: Select whether the specialized pricing will fall under
+  :guilabel:`Discount`, :guilabel:`Formula`, or :guilabel:`Fixed Price`. Depending on the price
+  type, there will be additional configurations on how to apply the pricelist.
 
-To add a specific product and price to a pricelist form, click the :guilabel:`Price Rules` tab, then
-click :guilabel:`Add a line` in the :guilabel:`Products` column. Then, select the desired product
-for which a specific price should be applied.
+  - :guilabel:`Discount`: Enter the percentage to be discounted. A mark-up can be configured by
+    using a negative value in this field.
 
-Next, if necessary, select a product variant under the :guilabel:`Variants` column (e.g. a specific
-product size, color, etc.). If no variants are selected, then this price will apply to all variants
-of the product.
+  - :guilabel:`Formula`: Calculate the pricelist rules based the following configuration:
 
-If a minimum amount of the product must be purchased in order to apply the specific price, enter the
-amount under the :guilabel:`Min. Quantity` column.
+    - :guilabel:`Discount`: Percentage discount to be applied.
+    - :guilabel:`Round off to`: Numerical value to act as round-off multiple, to be applied after
+      discount. The rounding method sets the price so that it is a multiple of the value in this
+      field.
 
-To configure the price of the product for this specific pricelist, enter the desired amount under
-the :guilabel:`Price` column. Then, there is the option to add a :guilabel:`Start Date` and
-:guilabel:`End Date` to the configured product price, if desired.
+.. note::
+   Rounding is applied *after* the discount and *before* the extra fee.
+
+    - :guilabel:`Extra Fee`: Fixed amount to be added or subtracted once :guilabel:`Discount` and
+      :guilabel:`Round off to` have been applied.
+
+  - :guilabel:`Fixed Price`: Enter fixed price for this pricelist. When applied, all product lines
+    in the quotation form will be updated to this price.
```

Comment:
It appears this note should be nested under the "Round off to" bullet, please indent and fix the note admonition bullet formatting :)
```suggestion
      .. note::
         Rounding is applied *after* the discount and *before* the extra fee.

         - :guilabel:`Extra Fee`: Fixed amount to be added or subtracted once :guilabel:`Discount`
           and :guilabel:`Round off to` have been applied.
         - :guilabel:`Fixed Price`: Enter fixed price for this pricelist. When applied, all product
           lines in the quotation form will be updated to this price.
```